go/libraries/doltcore/sqle/enginetest/dolt_transaction_commit_test.go

```diff
@@ -200,6 +200,22 @@ func TestDoltTransactionCommitTwoClients(t *testing.T) {
 				Query:    "/* client b */ START TRANSACTION;",
 				Expected: []sql.Row{},
 			},
+			// Concurrent with the two transactions which are going to (dolt_)commit changes, we
```

Comment:
This could use at least one equivalent test with manual `call dolt_commit` in `dolt_transaction_queries.go`